### DISCUSSION TITLE
Welcome to rha-rhcsa-challenge-excercises Discussions!

### DISCUSSION THREAD

**Discussion by wurquhar** [RedHatQuickCourses/rha-rhcsa-challenge-excercises#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wurquhar** [comment]
Testing this out as a thread for questions - can y'all see this?

**Reply by cbonhorst** [reply]
yup

**Reply by DNAtsol** [reply]
I see it!!
